CMakeLists.txt

```diff
@@ -1112,3 +1112,14 @@ FILE(STRINGS "include/openssl/opensslv.h"
 if(NOT ${BASE_VERSION_NUMBER} MATCHES ${OPENSSLV_VERSION_NUMBER})
   message( FATAL_ERROR "OPENSSL_VERSION_NUMBER in base.h and opensslv.h should match.")
 endif()
+
+# TODO: Find a way to make this automatically align with OPENSSL_VERSION_NUMBER in base.h.
+set(VERSION 1.1.1)
```

Comment:
OPENSSL_VERSION_NUMBER is [currently](https://github.com/aws/aws-lc/blob/main/include/openssl/base.h#L103) a hex integer `0x1010107f`. how does that translate to this version number here?

Reply:
This value should be an OpenSSL version recognizable by users.  For example, on my AmazonLinux host it looks like this:
```
❯ cat /usr/lib64/pkgconfig/libcrypto.pc | grep Version
Version: 3.0.8
```

Reply:
This translates to 1.1.1. There's more information on what the trailing `07f` means here:
https://github.com/openssl/openssl/blob/OpenSSL_1_1_1-stable/include/openssl/opensslv.h#L17-L42

There isn't an easy way to translate the string over from the header file to the CMake file, so I've left it as a TODO for now. The current method parses over to `Version: 1.1.1` format that Justin shared. The trailing numbers aren't shown in the final `pc` files.

Reply:
More context on how OpenSSL gets the version number. They use the configure script to parse the numbers from the `opensslv.h` file:
https://github.com/openssl/openssl/blob/OpenSSL_1_1_1-stable/Configure#L263-L276